wectl/wectl/commands/get.py

```
@@ -0,0 +1,77 @@
+@click.option("-n", "--namespace", default=None, help="Namespace (default: from config)")
+@click.option("-o", "--output", type=click.Choice(["wide", "yaml", "json"]), help="Output format")
+@click.option("-A", "--all-namespaces", is_flag=True, help="List from all namespaces")
+@click.option("--filter", "name_filter", help="Filter by name (partial match)")
+@click.pass_context
+def get_cmd(
+    ctx: click.Context,
+    kind: str,
+    name: Optional[str],
+    namespace: Optional[str],
+    output: Optional[str],
+    all_namespaces: bool,
```

Ruff (0.14.6)

Unused function argument: `all_namespaces`

(ARG001)